### DISCUSSION TITLE
How to use TikTokUser().get_insight() ?

### DISCUSSION THREAD

**Discussion by dwissaaj** [davidteather/TikTok-Api#730] [Q&A] [answered]
When I read the doc there is a function get_insights() at TikTokApi.tiktokuser. But there is no example code or tutorial so I am really confused, especially about parameter user_cookie. Does anyone have code example or where is to get user_cookie?.Thanks

**Comment by davidteather** [comment, marked as answer]
should be under
```
user_info = await api.user(username="someusername").info()
```
